openshift.spec

```diff
@@ -132,9 +121,6 @@ do
     go install -ldflags "%{ldflags}" %{import_path}/cmd/openshift
 done
 
-# set the IMAGES
-sed -i 's|IMAGES=.*|IMAGES=%{docker_images}|' rel-eng/openshift-{master,node}.sysconfig
-
```

Comment:
With the other changes, the services will choke without a config file anyway. I guess I don't really see a problem with that, although maybe we should supply config files with some workable defaults so that the services will start out of the box? Given we expect to be using the ansible installer, and the rate of change of config, not sure we want that yet though.

I guess LGTM as-is.